content/en/integrations/guide/prometheus-metrics.md

```diff
@@ -4,68 +4,111 @@ kind: guide
   text: "Kubernetes Prometheus and OpenMetrics metrics collection"
 ---
 
-If you are using Datadog's Prometheus or OpenMetrics checks, you may be interested in how these metrics map to existing Datadog metrics types.
+## Overview
 
-See [Kubernetes Prometheus and OpenMetrics Metrics Collection][1] for more information.
+This page walks you through how Prometheus or OpenMetrics check metrics map to existing Datadog metric types.
 
 ## Prometheus and OpenMetrics metric types
 
-* `counter`: a cumulative metric that represents a single monotonically increasing counter, whose value can only increase—or be reset to zero.
-* `gauge`: a metric that represents a single numeric value, which can arbitrarily go up and down.
-* `histogram`: samples observations and counts them in configurable buckets; also provides a sum of all observed values.
-* `summary`: similar to histogram; samples observations, provides a sum of all observed values, and calculates configurable quantiles over a sliding time window.
+* `counter`: A cumulative metric that represents a single monotonically increasing counter, whose value can only increase—or be reset to zero.
+* `gauge`: A metric that represents a single numeric value, which can arbitrarily go up and down.
+* `histogram`: Samples observations and counts them in configurable buckets; also provides a sum of all observed values.
+* `summary`: Similar to `histogram`; samples observations, provides a sum of all observed values, and calculates configurable quantiles over a sliding time window.
 
 ## How Prometheus/OpenMetrics metrics map to Datadog metrics
 
-For more information, see [Datadog Metric Types][2].
+For more information, see [OpenMetrics Metric Types][2] and [Datadog Metric Types][3].
+
+{{< tabs >}}
+{{% tab "Latest Version" %}}
+### Gauge
+
+[Prometheus/OpenMetrics `gauge`][101] maps to Datadog's `gauge`.
 
 ### Counter
 
-By default, [Prometheus/OpenMetrics `counter`][3] is mapped to Datadog's `monotonic_count`.
+[Prometheus/OpenMetrics `counter`][102] is mapped to Datadog's `count`.
+
+### Histogram
+
+For [Prometheus/OpenMetrics `histogram`][104], the `_count`, `_sum`, and `_bucket` values of the histogram are each mapped to Datadog's `count` type and include a `.count`, `.sum`, and `.bucket` suffix in their name, respectively.
 
-However, if the parameter `send_monotonic_counter` is `false`, then this metric is sent as `gauge`. [Read more about monotonic counters][4].
+If the `histogram_buckets_as_distributions` parameter is `true`, `_bucket` samples are aggregated into a Datadog `distribution`. [Datadog distribution metrics][108] are based on the [DDSketch algorithm][109] and allow for more advanced statistical aggregations such as quantiles. For more information, see the Datadog Engineering Blog [post on OpenMetrics and distribution metrics][105].
+
+`collect_counters_with_distributions` can be used to send `_count` and `_sum` values as `count`s alongside the distribution.
+
+### GaugeHistogram
+
+TBD.
```

Comment:
Did you mean to fill this in, or are you still waiting for information?

Reply:
Still waiting on info from @alopezz on this one!

Reply:
I don't think this applies, so we should just remove the section I believe.